Chirp/Pages/RegistrationPage/RegistrationViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RegistrationViewModel.swift
+//  Chirp
+//
+//  Created by Stepan Kukharskyi on 6/9/23.
+//
+
+import UIKit
+import Combine
+import FirebaseAuth
+
+class RegistrationViewModel {
+    
+    private var phoneNumber: String
+    private var imageExtension: String = "jpg"
+    
+    private let service = NetworkService.shared
+    private var cancellables: Set<AnyCancellable> = []
+    var didRegisterUser = PassthroughSubject<Bool,Never>()
+    
+    init(phoneNumber: String) {
+        self.phoneNumber = phoneNumber
+    }
+    
+    
+    func createAccount(name: String, profileImage: UIImage?) {
+        guard let userID = Auth.auth().currentUser?.uid else {
+            AlertToast.showAlert(message: AppError.registrationFailed.errorDescription ?? "Error", type: .error)
```

Comment:
You can also use `AppError.registrationFailed.localizedDescription` which returns a String instead of String?